pkgs/development/tools/database/pgweb/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildGoPackage, fetchFromGitHub, lib }:
```

Comment:
A question: is `lib` required here? If yes, is `maintainers` required too? If not, why not?

Reply:
lib is exposed as a top-level "package" so injection will work for it, maintainers, platforms and so on are nested fields, so they can't be injected by callPackage.

So yes you inject lib, and then you use it to build the meta field (and potentially to call other functions)

Reply:
So it's good like it is now?